src/components/Kv/KvButton.vue

```diff
@@ -75,6 +91,10 @@ export default {
 		}
 	}
 
+	&.alert {
+		@include alert-styles();
```

Comment:
Nice, this alert button will be useful in more places I'm sure.
There's a handy Foundation mixin that can can replace your alert-styles mixin and adds darker hover colors like the default button (but still have to include box shadow).
```
	&.alert {
		@include button-style($kiva-accent-red, auto, $white);
		box-shadow: $button-secondary-shadow darken($kiva-accent-red, 10%);
	}
```

Reply:
Thanks @ryan-ludwig this does look way better 👍